README.md

```diff
@@ -36,13 +36,15 @@ Get the tool via
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+Typical usage - either on a new project or converting an existing one - is
 
 ```sh
 $ dep init
 $ dep ensure -update
 ```
 
+If your project was already using a `vendor/` directory, it was backed up to `_vendor-TIMESTAMP/`.
```

Comment:
"If a `vendor/` directory existed in your project root, it should have been backed up to `_vendor-TIMESTAMP/`."?

I'm not sure, but I think this one could use some rephrasing.